src/WebJobs.Extensions.DurableTask/Correlation/TraceContextBaseExtensions.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using DurableTask.Core;
+#if !FUNCTIONS_V1
```

Comment:
You may want to extend this conditional compilation up, as it appears none of these imports are necessary in Functions V1.

Reply:
In fact, for many of these correlation files, it may be eaiser to conditionally exclude them from Functions V1 builds in the csproj file: https://stackoverflow.com/questions/14175853/c-is-it-possible-to-conditionally-exclude-some-files-from-compilation.

Reply:
Done